Bio/SeqIO/QualityIO.py

```diff
@@ -796,6 +796,59 @@ def _get_solexa_quality_str(record):
                    for qp in qualities)
 
 
+def FastqStrictIterator(handle):
+    """Iterate over strictly four line Fastq records as string tuples.
+
+    In contrast to the FastqGeneralIterator, this iterator does allow for the
+    splitting of records over multiple lines.  Line wrapping is out of vogue and
+    hence this iterator provides a faster but stricter alternative to the
+    FastqGeneralIterator.
+
+    Iterate over Fastq records as string tuples (not as SeqRecord objects).
+
+    This code does not try to interpret the quality string numerically.  It
+    just returns tuples of the title, sequence and quality as strings.  For
+    the sequence and quality, any whitespace (such as new lines) is removed.
```

Comment:
There won't be any new line white space (except in the corner case of a zero length sequence).

Reply:
Yeah, sorry, I just used the base documentation from FastqGeneralIterator. I've edited the doc for this. Should we also edit this out in FastqGeneralIterator?